src/argilla/client/feedback/dataset/local.py

```diff
@@ -34,7 +36,7 @@ def __init__(
         self,
         *,
         fields: List["AllowedFieldTypes"],
-        questions: List[AllowedQuestionTypes],
+        questions: List["AllowedQuestionTypes"],
```

Comment:
No need to use double-quotes as it's not under the `TYPE_CHECKING`

```suggestion
        questions: List[AllowedQuestionTypes],
```